templates/ehub-new-setup.json

```diff
@@ -48,9 +52,8 @@
         "tenantId": "[subscription().tenantId]",
         "contributor": "[concat('/subscriptions/', subscription().subscriptionId, '/providers/Microsoft.Authorization/roleDefinitions/', 'b24988ac-6180-42a0-ab88-20f7382dd24c')]",
```

Comment:
Maybe worth renaming to contributorRole to keep it consistent with the other template?